mitx/lms/templates/courseware/course_about.html

```diff
@@ -191,7 +194,12 @@ <h1>
             else:
               href_class = "register"
           %>
-          <a href="${reg_href}" class="${href_class}">
+          <a
+            href="${reg_href}"
+            class="${href_class}"
+            data-toggle="tooltip"
+            title="Enroll in the course to take advantage of advanced assessments and keep track of course progress."
```

Comment:
nit:
```suggestion
            title="Enroll in the course to take advantage of advance assessments and keep track of your course progress."
```

Reply:
Got the content from the client themself.